tasks/files/snapshot.py

```diff
@@ -1153,8 +1039,12 @@ def umount_snapshot_set(snapset_json, verify_only, check_mode):
 
     changed = False
     for list_item in volume_list:
-        vg_name = list_item["vg"]
-        lv_name = list_item["lv"]
+        if "vg" in list_item:
+            vg_name = list_item["vg"]
```

Comment:
else - `vg_name` will either be undefined, in which case you will get a `NameError` if you try to use it - or it will be using the value from a previous loop iteration, which will likely lead to weird behavior

```python
            return (
                SnapshotStatus.ERROR_UMOUNT_VERIFY_FAILED,
                "set item must provide a mountpoint for : " + vg_name + "/" + lv_name,
                changed,
            )
```

Reply:
I'll move any checks for "vg" and "lv" to the validate_json_umount() since they are required parameters.  Ok?